src/components/HomePage/Forms/SelectInputGenre.tsx

```diff
@@ -1,39 +1,50 @@
-import React, { FC } from 'react';
+import React, { FC, useRef } from 'react';
 import {
-  FormGroup, Input, Label,
+    Input, Label,
 } from 'reactstrap';
 import './Form.scss';
+import { Field, useField } from "formik";
 
-export const SelectInputGenre: FC<{ defaultSelected?: string[],
-    onChangeCallBack: Function }> = ({ defaultSelected, onChangeCallBack }) => {
-  const movieGenres = ['Action', 'Adventure', 'Animation',
-    'Comedy', 'Drama', 'Family',
-    'Fantasy', 'Horror', 'Music',
-    'Mystery', 'Romance', 'Science Fiction',
-    'Thriller'];
+export const SelectInputGenre: FC<{
+    defaultSelected: string[],
+    id: string
+}> = ({ defaultSelected, id }) => {
+    const inputRef = useRef<HTMLInputElement | null>(null);
+    const movieGenres = ['Action', 'Adventure', 'Animation',
+        'Comedy', 'Crime', 'Documentary',
+        'Drama', 'Family', 'Fantasy',
+        'Horror', 'Music', 'Mystery',
+        'Romance', 'Science Fiction', 'Thriller'];
+    const genresName = "genres";
```

Comment:
and this one to constants.ts

Reply:
Done in commit 144e7682797a8b327df9c556453f21aad1d10993